db-drda/src/main/java/com/splicemachine/db/impl/drda/DRDAConnThread.java

```diff
@@ -4549,7 +4551,13 @@ private boolean parseEXCSQLSTTobjects(DRDAStatement stmt) throws DRDAProtocolExc
 		
 		if (! gotEXTDTA) {
 			stmt.ps.clearWarnings();
-			result = stmt.execute();
+			if (updateCounts.length > 1) {
+				int[] results = stmt.executeBatch();
```

Comment:
For batch execution, the returned value result is not set, so it goes with the default value of false, is this desired behavior?

Reply:
Also at line 4534 below:
					case CodePoint.EXTDTA:	
						readAndSetAllExtParams(stmt, true);
						stmt.ps.clearWarnings();
						result = stmt.execute();   <--- line 4534
						gotEXTDTA = true;
						break;
It handles execution of single statement, do we need to worry about executeBatch() here?

Reply:
result should be false, yes, since batched execution is only for DML statements that don't have a result set.

Regarding line 4534, I'm not sure what role that plays or when that gets executed. We could add logic to handle batched execution here but I'm not sure how to test it.